src/app/service/user.service.ts

```diff
@@ -0,0 +1,105 @@
+import {Injectable} from '@angular/core';
+import {User} from "../domain/User";
+import {HttpClient, HttpHeaders, HttpResponse} from "@angular/common/http";
+import {Router} from "@angular/router";
+import {catchError, Subject, throwError} from "rxjs";
+import {mapStringToUserRole, UserRoles} from "../util/enum";
+import {jwtDecode} from "jwt-decode";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  private uri = 'http://localhost:9080/yea-backend/users'
```

Comment:
Het lijkt me handig om  'http://localhost:9080/yea-backend'  als backendUrl in environment.development.ts te zetten, en dan daarnaar te verwijzen.